docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,37 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger API
+~~~~~~~~~~
+- **BREAKING** Removed the unused field :ref:`com.digitalasset.ledger.api.v1.ExercisedEvent` from :ref:`com.digitalasset.ledger.api.v1.Event`.
+  ``Event`` is only used in :ref:`com.digitalasset.ledger.api.v1.Transaction`, which in turn by definition never contains exercised events (only created and archived events): `#960 <https://github.com/digital-asset/daml/issues/960>`_
```

Comment:
I'm not sure if it needs the issue link? It seems clear enough to me.

Reply:
Francesco asked us to always link to the respective issues.